vsts/pipelines/ci.yml

```diff
@@ -1,4 +1,10 @@
 parameters:
+  - name: storageAccountUrl
+    displayName: Storage Account Url
```

Comment:
nit: if we can use this `displayName` property as a "description", it would be nice to include some additional information about how this URL is being used

_e.g._, "Storage Account URL for Production Images and Testing"